daemon/meson.build

```diff
@@ -0,0 +1,59 @@
+# Machine Learing Agent
+if get_option('enable-machine-learning-agent')
+  nns_ml_agent_srcs = []
+  nns_ml_agent_incs = []
+  nns_ml_agent_incs += include_directories('includes')
+
+  # Generate GDbus header and code
+  gdbus_prog = find_program('gdbus-codegen')
+  if not gdbus_prog.found()
+    error('gdbus_codegen is not found!')
+  endif
```

Comment:
```suggestion
  gdbus_prog = find_program('gdbus-codegen', required : true)
```

Reply:
Simple one line is easy to understand. Applied!